### PR TITLE
Several small doc changes and robins PR merged

### PR DESCRIPTION
**Pull request by nitrosimon**
PR from Robin merged with fixed merge conflicts and several other smaller changes

### CODE REVIEW

File: start/factory-reset.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    $ nitropy start set-identity 0`
+    $ gpg --card-edit` -> admin -> factory-reset
+
+It deletes all keys, pins, configurations and meta data associated with the respective identity.
+
+
+Reset code
+----------
+
+This step is optional for new firmware.. You can add an reset code which enable the reset of the User PIN (not the admin PIN). For older Firmware it is recommended since it is needed to reset the device (see below)
```
Comment (jans23) [inline start/factory-reset.rst:62]: "an reset code" -> "a reset code"

**Comment by jans23** [comment]
I believe you don't want to upload test.txt, don't you?

File: start/factory-reset.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
+Factory Reset
+===================
```
Comment (jans23) [inline start/factory-reset.rst:2]: wrong length

File: start/factory-reset.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+This step is optional with new firmware.. You can add a reset code which enable the reset of the User PIN (not the Admin PIN). For older Firmware it is recommended since it is needed to reset the device (see below)
+
+
+`Checkout the documentation here <http://www.fsij.org/doc-gnuk/gnuk-passphrase-setting.html#set-up-pw1-pw3-and-reset-code>`_.
+
+
+
+
+Firmware below 1.2.5
+------
```
Comment (jans23) [inline start/factory-reset.rst:71]: wrong length here and in future lines.

**Comment by daringer** [comment]
meeergin now, nice!

**Pull request merged**